tests/MockRiscZeroVerifier.sol

```diff
@@ -49,14 +41,13 @@ contract RiscZeroGroth16VerifierTest is IRiscZeroVerifier {
     ) public view returns (bool) {
         // Require that the seal be specifically empty.
         // Reject if the caller may have sent a real seal.
-        return CONTROL_ID_0 == 0 && CONTROL_ID_1 == 0 && seal.length == 0 && postStateDigest == bytes32(0);
```

Comment:
I've added those to remove all the warnings from the solidity compiler. e.g.,:
```
Warning (2018): Function state mutability can be restricted to pure
  --> tests/MockRiscZeroVerifier.sol:35:5:
   |
35 |     function verify(
   |     ^ (Relevant source part starts here and spans across multiple lines).
```

Reply:
Weird, I didn't see those errors until I did `git clean` to clear my cache.